editions/tw5.com/tiddlers/widgets/RevealWidget.tid

```diff
@@ -87,4 +87,24 @@ Here is a simple example how to use text references with field content to contro
 
 <<wikitext-example-without-html '<$reveal type="match" state="!!jeremy" text="tiddlywiki">
 ~TiddlyWiki!
-</$reveal>'>>
\ No newline at end of file
+</$reveal>'>>
+
+
+!! How to compare values stored in variables
+
+By using the ``default`` parameter instead of ``state``, variables can be compared as seen in these two examples:
+
+
+```
+<$set name="variable" value="somevalue"><$reveal text=<<variable>> type="match" default="somevalue">
+this is revealed text
+</$reveal> </$set>
+```
+
+or
```

Comment:
I would think one example is fine, since it is a general property of attributes that one can define them using variables or text references.